libfuse/file.go

```diff
@@ -247,29 +247,6 @@ func (f *File) Write(ctx context.Context, req *fuse.WriteRequest,
 	return nil
 }
 
-var _ fs.HandleFlusher = (*File)(nil)
-
-// Flush implements the fs.HandleFlusher interface for File.
-func (f *File) Flush(ctx context.Context, req *fuse.FlushRequest) (err error) {
```

Comment:
Correct.  But "flush" isn't called by an application directly, it's just what FUSE calls on on a `Close` operation (https://godoc.org/bazil.org/fuse#FlushRequest).  As far as I know, it's optional.